go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/yeqown/fasthttp-reverse-proxy/v2
 
 require (
-	github.com/fasthttp/websocket v1.4.1
-	github.com/stretchr/testify v1.6.1 // indirect
-	github.com/valyala/fasthttp v1.4.0
-	github.com/yeqown/log v1.0.5
+	github.com/andybalholm/brotli v1.0.3 // indirect
```

Comment:
remove indirect dependency

Reply:
That's added by go mod so removing it might not be good idea

Reply:
which version of Go are you using?

Reply:
`go version go1.17.1 linux/amd64`

Reply:
And I tried to upgrade the version of `fasthttp`, go mod would not add these two packages too ~

Reply:
Hard to tell, did you run go mod tidy?

Reply:
yeah